src/polymer-project.ts

```diff
@@ -244,6 +222,69 @@ class HtmlSplitter extends Transform {
       callback(null, file);
     }
   }
+
+  _splitScript(file: File, doc: parse5.ASTNode, filePath: string) {
+    const scriptTags = dom5.queryAll(doc, HtmlSplitter.isInlineScript);
+    for (let i = 0; i < scriptTags.length; i++) {
```

Comment:
you can use `return` in the `forEach()` iterator to achieve the same behavior as `continue`